core/templates/domain/objects/NumberWithUnitsObjectFactory.ts

```diff
@@ -68,12 +68,12 @@ export class NumberWithUnits {
   }
 
   toString(): string {
-    var numberWithUnitsString: string = '';
+    let numberWithUnitsString = '';
```

Comment:
Why removing the predefined types here ?

Reply:
iamprayush [asked to remove](https://github.com/oppia/oppia/pull/13369#discussion_r671797298) these types above because these can be inferred, so I removed them from here as well.